aspnetcore/fundamentals/servers/kestrel.md

```diff
@@ -561,7 +562,27 @@ In the following *appsettings.json* example:
 }
 ```
 
-An alternative to using **Path** and **Password** for any certificate node is to specify the certificate using certificate store fields. For example, the **Certificates** > **Default** certificate can be specified as:
+Schema notes:
+
+* Endpoints names are case-insensitive. For example, `HTTPS` and `Https` are valid.
+* The `Url` parameter is required for each endpoint. The format for this parameter is the same as the top-level `Urls` configuration parameter except that it's limited to a single value.
+* These endpoints replace those defined in the top-level `Urls` configuration rather than adding to them. Endpoints defined in code via `Listen` are cumulative with the endpoints defined in the configuration section.
```

Comment:
Needs an example. I know what it means, but would all readers?

Reply:
Just checking, is it literally `Urls` or is it the plural of `Url`? This needs to be a literal and you can't make literals plural by tacking on a `s`